.github/workflows/actions.yml

```
@@ -39,6 +53,12 @@ jobs:
           go mod tidy
           go mod download
 
+      - name: Set environment variables for Elasticsearch
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)